### ISSUE TITLE
Signs shouldn't be four directional

### ISSUE THREAD

**Issue by ElectroJr** [space-wizards/space-station-14#7187]
While this makes it super convenient for mappers, it causes issues when looking at the station from a rotated viewpoint. In particular, when there is more than one sign on the same bit of wall, at a 90 degree angle they will start overlapping. The directions should probably just be different states & there need to be different prototypes for each direction. I guess this also raises the issue of whether the arrow should be a separate layer, which was also discussed a bit in #6401.

Basically, the sign in this video shouldn't be flipping around and should just be upside down at times.

https://user-images.githubusercontent.com/60421075/159105322-30cc3967-e124-4f3e-9ba5-d9025f285b47.mp4

**Comment by GoodDays13** [comment]
i feel like they rotate their positon on the tile when the view rotates so they don't overlap

**Comment by IProduceWidgets** [comment]
What would really be helpful is a system for placing multiple signs on the same wall and having them align both with the wall, and with eachother.

**Comment by GoodDays13** [comment]
Maybe a solution could be having an actual location (just placed in the center of the wall tile on this instance) then a visual offset value that is relative to the camera's orientation rounded to the nearest cardinal direction of the wall it's placed on. The offset would be stored server-side, but the interpretation of how to render it would be client-side.

**Comment by ProPeperos** [comment]
could be easily fixed by rotating the east and west sprites sideways
